script/BaseScript.s.sol

```diff
@@ -39,11 +41,15 @@ struct Deployment {
 struct ChainDeployment {
     uint256 chainId;
     address erc20Address;
+    address erc20ProxyAdmin;
+    address erc20Timelock;
```

Comment:
let's use the same timelock for this system we don't need a timelock for each component and this way we can do atomic upgrades easier